### PR TITLE
Add Container Signing

### PR DESCRIPTION
**Pull request by rochacbruno**
Issue: AAH-1358

#### What is this PR doing:

Enabled the workflows described on https://hackmd.io/@rochacbruno/SkQmp8-09

#### Reviewers must know:

The config added here affects only the ./compose based dev environment, other installer will be addressed separatelly

#### Related

- Issue opened on pulp_installer https://github.com/pulp/pulp_installer/issues/1347

### CODE REVIEW

**Comment by MilanPospisil** [comment]
Signing for local container works, next I will try remote containers.

File: galaxy_ng/app/access_control/statements/pulp_container.py [hunk 1]
```diff
@@ -155,7 +155,7 @@
                 "condition": "has_namespace_or_obj_perms:container.view_containerpushrepository",
             },
             {
-                "action": ["tag", "untag", "remove_image"],
+                "action": ["tag", "untag", "remove_image", "sign", "remove_signatures"],
```
Comment (newswangerd) [inline galaxy_ng/app/access_control/statements/pulp_container.py:158]: can you sign container repositories too, or just container push?

**Review by newswangerd — APPROVED** [review]
LGTM

File: galaxy_ng/app/access_control/statements/pulp_container.py (hunk 1 above)
Comment (MilanPospisil) [reply galaxy_ng/app/access_control/statements/pulp_container.py:158]: Yes, container repositories works too.

**Pull request merged**